examples/ui-demo/src/components/configuration/Authentication.tsx

```diff
@@ -106,7 +107,21 @@ export const Authentication = ({ className }: { className?: string }) => {
                     icon={<FacebookLogo />}
                     onClick={setAddFacebookAuth}
                   />
-                  {/* TO DO: add "Add provider" button */}
+                  <Button className="active:bg-[#EFF4F9]">
+                    <ExternalLink href="https://accountkit.alchemy.com/signer/authentication/auth0">
+                      <div className="flex flex-1 min-w-full flex-row justify-between gap-3 items-center">
+                        <p className="font-normal text-sm text-secondary-foreground">
+                          Add provider
```

Comment:
let's actually make this custom https://alchemyinsights.slack.com/archives/C07JXHYK8TT/p1728604393804589?thread_ts=1728604191.009179&cid=C07JXHYK8TT